controllers/kubeadmconfig_controller.go

```diff
@@ -52,13 +52,19 @@ const (
 type KubeadmConfigReconciler struct {
 	client.Client
 	SecretsClientFactory SecretsClientFactory
+	KubeadmInitLock      InitLocker
```

Comment:
I might be missing it, but where is this used? It looks like we make a second call to NewInitLocker below.

Reply:
`>.>` you didn't see anything

![](http://giphygifs.s3.amazonaws.com/media/4pMX5rJ4PYAEM/giphy.gif)